WcaOnRails/lib/custom_i18n_scanner.rb

```diff
@@ -0,0 +1,76 @@
+require 'i18n/tasks/scanners/file_scanner'
+class WCAFileScanner < I18n::Tasks::Scanners::FileScanner
+  include I18n::Tasks::Scanners::RelativeKeys
+  include I18n::Tasks::Scanners::OccurrenceFromPosition
+
+  def active_record?(model)
+    # FIXME: Find a way to determine this dynamically ?
+    model != "contact"
+  end
+
+  def extract_model_attr(key)
+    model = key.gsub(/^(\w+)\..*$/, '\1').singularize
+    # Some dirty tricks to assign the detected field to the "correct" model
+    if model == "devise"
+      model = "user"
+    elsif model == "admin"
+      model = "person"
+    elsif model == "oauth"
+      model = "doorkeeper/application"
+    end
```

Comment:
It would be nice to have short comment explaining this in the code.
And just wondering, that could be problematic if we would be to edit another model in the admin section, right? Fortunatelly it's not the case at the moment.